cloudy_launcher.py

```diff
@@ -0,0 +1,110 @@
+'''
+Simulates the launcher for CloudyGame
+
+Starts a loop that does: 
+1. get user input:
+       join <username>
+       quit <username>
+       exit
+2. send join/quit command to CloudyPanel
+3a. receive controllerId if success.
+    start thin client with given controllerId.
+3b. receive error message if fail.
+
+'''
+
+import socket
+import errno
+import os
+import thin_client
+
+
+JOIN = "join"
+QUIT = "quit"
+EXIT = "exit"
+COMMANDS = "Commands: \n" + \
+           "connect and join user -- join <username> \n" + \
+           "connect and quit user -- quit <username> \n" + \
+           "exit cloudy_launcher  -- exit \n"
+
+
+if __name__ == "__main__":
```

Comment:
wrap every logic inside a function. thus you just call:

    if __name__ == "__main__":
        main()